pkg/scheduler/objects/node.go

```diff
@@ -27,6 +27,7 @@ import (
 	"go.uber.org/zap"
```

Comment:
General comment: we don't have event change details for placeholder replacement and reservation/unreservation. I don't know if we need those or not, I assume the replace would be useful.

Reply:
Placeholders: no that is just another allocation for the node. Nothing special from that side.
Reservation: file a follow up jira to look at that. We use a reservation in different ways specially in preemption. We need to think where we report it (app or node) and how we report it.

Reply:
Created the follow-up.